frontend/src/components/Auth/Login/Login.js

```diff
@@ -14,9 +13,10 @@ async function handleSubmit(params, setStatus) {
   formData.append("password", params.data.password);
   (async () => {
     try {
-      await ky.post("http://127.0.0.1:8000/login", {
+      await ky.post("/api/login", {
         body: formData
       });
+      window.location.reload();
```

Comment:
This is a red flag and indicates a bad design pattern. What do you need this for?